index.js

```diff
@@ -551,6 +542,13 @@ function fixNonStringValue(value) {
     return value;
   }
 
+  if (value === 'True') {
```

Comment:
`true` is handled by `JSON.parse`.
But you totally right about `TRUE` case.
As you suggested, I will downcase string before comparation.